1-perfect-hashing/perfect_hashing.cpp

```diff
@@ -0,0 +1,143 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+typedef unsigned int uint;
+
+const int PRIME=41,MAXN=37,MAXP=55;//MAXN - maximal count of teams, MAXP - maximal count of players in the team
```

Comment:
Make constants more informative instead of comment them. When you want to comment some constants or variables then you better rename them.

Reply:
Already renamed:)